tests_integration/test_config.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from snowflake.connector.compat import IS_WINDOWS
+
+
+@pytest.mark.integration
+def test_config_file_permissions_warning(runner, recwarn):
```

Comment:
Maybe just skip if not windows?

Reply:
We want to test that there is no warning on windows